test_spack.py

```diff
@@ -295,6 +295,27 @@ def test_install_exclaim_cpu_gcc(self):
             'icon@exclaim-master%gcc icon_target=cpu +eccodes +ocean')
 
 
+class InferoTest(TestCase):
+    package_name = 'infero'
+    depends_on = {}
+    #    'eckit','fckit','tensorflowc'
```

Comment:
Only spack-c2sm internal dependencies are needed. Please remove the commented-out code.